### DISCUSSION TITLE
There are a lot of parameters for ActivityInput which aren't documented - Would any of them allow us to mark a property as required?

### DISCUSSION THREAD

**Discussion by sjd2021** [elsa-workflows/elsa-core#4264] [Q&A]
I don't want an activity to kick off if some basic required inputs aren't filled in. I prefer to not have to manually validate them at too low a level, IMO, in the workflow execution pipeline. I asked ChatGPT if it could give me a hint (it's been helpful before), and it totally made up some `bool Required` property that doesn't exist.. lol

**Comment by sfmskywalker** [comment]
Hello @sjd2021,

Elsa currently lacks comprehensive validation support, but rest assured, it's on our roadmap for Elsa 3. And hey, if ChatGPT starts offering solutions, maybe we should nudge it to submit some PRs too! Who knows, it might just surprise us one day. 😉
